[2024-03-14 12:24:51,714: INFO: main: Logging from main.py]
[2024-03-18 14:36:01,081: INFO: common: yaml file: /home/varun/Documents/TextSummarization/src/textSummarizer/config/configuration.py loaded successfully]
[2024-03-18 14:36:56,492: INFO: common: yaml file: /home/varun/Documents/TextSummarization/src/textSummarizer/config/configuration.py loaded successfully]
[2024-03-18 14:37:30,338: INFO: common: yaml file: /home/varun/Documents/TextSummarization/config/config.yaml loaded successfully]
[2024-03-18 14:38:02,131: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-18 14:41:28,359: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-18 14:41:33,412: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-18 14:41:33,414: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-18 14:41:33,415: INFO: common: created directory at: artifacts]
[2024-03-18 14:41:33,416: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-18 14:41:35,279: INFO: 3541854288: artifacts/data_ingestion/summarizer-data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 9830:177EEB:1DC225:2920A7:65F80545
Accept-Ranges: bytes
Date: Mon, 18 Mar 2024 09:11:34 GMT
Via: 1.1 varnish
X-Served-By: cache-del21746-DEL
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1710753094.921843,VS0,VE705
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 18c9bbcce687f4a6f3239ba50e892e43fbf15a17
Expires: Mon, 18 Mar 2024 09:16:34 GMT
Source-Age: 0

]
[2024-03-18 14:43:57,679: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-18 14:43:57,681: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-18 14:43:57,682: INFO: common: created directory at: artifacts]
[2024-03-18 14:43:57,682: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-18 14:43:57,683: INFO: 1090604987: File already exists of size: ~7718 KB]
[2024-03-18 14:44:17,145: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-18 14:44:17,147: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-18 14:44:17,147: INFO: common: created directory at: artifacts]
[2024-03-18 14:44:17,148: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-18 14:44:17,149: INFO: 2459005936: File already exists of size: ~7718 KB]
[2024-03-18 14:52:06,156: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-18 14:52:06,158: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-18 14:52:06,158: INFO: common: created directory at: artifacts]
[2024-03-18 14:52:06,159: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-18 14:52:06,160: INFO: 2459005936: File already exists of size: ~7718 KB]
[2024-03-18 15:27:10,861: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-03-18 15:27:10,862: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-18 15:27:10,862: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-18 15:27:10,862: INFO: common: created directory at: artifacts]
[2024-03-18 15:27:10,862: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-18 15:27:10,863: ERROR: main: name 'DataIngestion' is not defined]
Traceback (most recent call last):
  File "/home/varun/Documents/TextSummarization/main.py", line 8, in <module>
    data_ingestion.main()
  File "/home/varun/Documents/TextSummarization/src/textSummarizer/pipeline/satge_01_data_ingestion.py", line 13, in main
    data_ingestion = DataIngestion(config=data_ingestion_config)
NameError: name 'DataIngestion' is not defined
[2024-03-18 15:33:15,177: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-03-18 15:33:15,178: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-18 15:33:15,179: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-18 15:33:15,179: INFO: common: created directory at: artifacts]
[2024-03-18 15:33:15,179: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-18 15:33:17,769: INFO: data_ingestion: artifacts/data_ingestion/summarizer-data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 9830:177EEB:1DC225:2920A7:65F80545
Accept-Ranges: bytes
Date: Mon, 18 Mar 2024 10:03:17 GMT
Via: 1.1 varnish
X-Served-By: cache-del21744-DEL
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1710756197.957097,VS0,VE283
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: dfd4c4c43d3a825b111c9eac15b78647201c7a41
Expires: Mon, 18 Mar 2024 10:08:17 GMT
Source-Age: 0

]
[2024-03-18 15:33:17,875: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-03-18 15:34:01,284: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-03-18 15:34:01,284: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-18 15:34:01,285: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-18 15:34:01,285: INFO: common: created directory at: artifacts]
[2024-03-18 15:34:01,285: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-18 15:34:01,285: ERROR: main: name 'DataIngesion' is not defined]
Traceback (most recent call last):
  File "/home/varun/Documents/TextSummarization/main.py", line 8, in <module>
    data_ingestion.main()
  File "/home/varun/Documents/TextSummarization/src/textSummarizer/pipeline/satge_01_data_ingestion.py", line 12, in main
    data_ingestion = DataIngesion(config=data_ingestion_config)
NameError: name 'DataIngesion' is not defined
[2024-03-18 15:34:25,320: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-03-18 15:34:25,321: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-18 15:34:25,321: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-18 15:34:25,321: INFO: common: created directory at: artifacts]
[2024-03-18 15:34:25,322: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-18 15:34:26,007: INFO: data_ingestion: artifacts/data_ingestion/summarizer-data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 9830:177EEB:1DC225:2920A7:65F80545
Accept-Ranges: bytes
Date: Mon, 18 Mar 2024 10:04:25 GMT
Via: 1.1 varnish
X-Served-By: cache-del21750-DEL
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1710756266.589639,VS0,VE1
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: ce4d8c5196915a3db299efdb70424e5138f8798b
Expires: Mon, 18 Mar 2024 10:09:25 GMT
Source-Age: 68

]
[2024-03-18 15:34:26,122: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-03-21 11:43:47,355: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-21 11:43:47,363: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 11:43:47,365: INFO: common: created directory at: artifacts]
[2024-03-21 11:43:47,366: INFO: common: created directory at: artifacts/data_validation]
[2024-03-21 11:46:01,596: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-21 11:46:01,598: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 11:46:01,598: INFO: common: created directory at: artifacts]
[2024-03-21 11:46:01,599: INFO: common: created directory at: artifacts/data_validation]
[2024-03-21 11:46:08,075: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-21 11:46:08,077: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 11:46:08,077: INFO: common: created directory at: artifacts]
[2024-03-21 11:46:08,078: INFO: common: created directory at: artifacts/data_validation]
[2024-03-21 11:47:20,881: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-21 11:47:20,882: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 11:47:20,883: INFO: common: created directory at: artifacts]
[2024-03-21 11:47:20,883: INFO: common: created directory at: artifacts/data_validation]
[2024-03-21 11:47:49,955: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-21 11:47:52,656: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-21 11:47:52,657: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 11:47:52,664: INFO: common: created directory at: artifacts]
[2024-03-21 11:47:52,667: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-21 11:47:57,414: INFO: 2459005936: artifacts/data_ingestion/summarizer-data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 7FC8:2D9B30:28AF21:34778B:65FBD110
Accept-Ranges: bytes
Date: Thu, 21 Mar 2024 06:17:54 GMT
Via: 1.1 varnish
X-Served-By: cache-maa10238-MAA
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1711001873.381133,VS0,VE708
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: b5e34bcc380daf315ab764ffbcb2a23cca927f99
Expires: Thu, 21 Mar 2024 06:22:54 GMT
Source-Age: 0

]
[2024-03-21 11:48:22,311: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-21 11:48:22,312: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 11:48:22,312: INFO: common: created directory at: artifacts]
[2024-03-21 11:48:22,313: INFO: common: created directory at: artifacts/data_validation]
[2024-03-21 11:48:33,215: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-21 11:48:33,217: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 11:48:33,217: INFO: common: created directory at: artifacts]
[2024-03-21 11:48:33,218: INFO: common: created directory at: artifacts/data_validation]
[2024-03-21 11:50:05,040: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-21 11:50:08,893: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-21 11:50:08,895: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 11:50:08,896: INFO: common: created directory at: artifacts]
[2024-03-21 11:50:08,896: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-21 11:50:15,859: INFO: 2459005936: artifacts/data_ingestion/summarizer-data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 7FC8:2D9B30:28AF21:34778B:65FBD110
Accept-Ranges: bytes
Date: Thu, 21 Mar 2024 06:20:09 GMT
Via: 1.1 varnish
X-Served-By: cache-maa10241-MAA
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1711002010.729417,VS0,VE1
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 07952ffaa3db355dce4a497b49f97d792c1ae864
Expires: Thu, 21 Mar 2024 06:25:09 GMT
Source-Age: 136

]
[2024-03-21 11:50:41,527: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-21 11:50:41,528: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 11:50:41,529: INFO: common: created directory at: artifacts]
[2024-03-21 11:50:41,529: INFO: common: created directory at: artifacts/data_validation]
[2024-03-21 12:13:33,453: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-03-21 12:13:33,454: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-21 12:13:33,455: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 12:13:33,455: INFO: common: created directory at: artifacts]
[2024-03-21 12:13:33,455: INFO: common: created directory at: artifacts/data_validation]
[2024-03-21 12:13:33,455: ERROR: main: local variable 'data_validation_config' referenced before assignment]
Traceback (most recent call last):
  File "/home/varun/Documents/TextSummarization/main.py", line 20, in <module>
    data_validation.main()
  File "/home/varun/Documents/TextSummarization/src/textSummarizer/pipeline/stage_02_data_validation.py", line 11, in main
    data_validation_config = config.get_data_validation_config()
  File "/home/varun/Documents/TextSummarization/src/textSummarizer/config/configuration.py", line 27, in get_data_validation_config
    data_validation_config = data_validation_config(
UnboundLocalError: local variable 'data_validation_config' referenced before assignment
[2024-03-21 12:15:21,863: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-03-21 12:15:21,864: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-21 12:15:21,865: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 12:15:21,865: INFO: common: created directory at: artifacts]
[2024-03-21 12:15:21,865: INFO: common: created directory at: artifacts/data_validation]
[2024-03-21 12:15:21,865: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/samsum_dataset']
Traceback (most recent call last):
  File "/home/varun/Documents/TextSummarization/main.py", line 20, in <module>
    data_validation.main()
  File "/home/varun/Documents/TextSummarization/src/textSummarizer/pipeline/stage_02_data_validation.py", line 13, in main
    data_validation.validate_all_files_exist()
  File "/home/varun/Documents/TextSummarization/src/textSummarizer/components/data_validation.py", line 24, in validate_all_files_exist
    raise e
  File "/home/varun/Documents/TextSummarization/src/textSummarizer/components/data_validation.py", line 12, in validate_all_files_exist
    all_files = os.listdir(os.path.join("artifacts", "data_ingestion", "samsum_dataset"))
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/samsum_dataset'
[2024-03-21 12:16:00,073: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-03-21 12:16:00,074: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-21 12:16:00,074: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 12:16:00,074: INFO: common: created directory at: artifacts]
[2024-03-21 12:16:00,074: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-21 12:16:04,270: INFO: data_ingestion: artifacts/data_ingestion/summarizer-data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 7FC8:2D9B30:28AF21:34778B:65FBD110
Accept-Ranges: bytes
Date: Thu, 21 Mar 2024 06:46:01 GMT
Via: 1.1 varnish
X-Served-By: cache-maa10237-MAA
X-Cache: HIT
X-Cache-Hits: 1
X-Timer: S1711003561.067131,VS0,VE269
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 9ab5c55280def805108a60e5200e6fb572495d35
Expires: Thu, 21 Mar 2024 06:51:01 GMT
Source-Age: 0

]
[2024-03-21 12:16:04,382: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-03-21 12:16:04,382: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-03-21 12:16:04,383: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-03-21 12:16:04,384: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 12:16:04,384: INFO: common: created directory at: artifacts]
[2024-03-21 12:16:04,384: INFO: common: created directory at: artifacts/data_validation]
[2024-03-21 12:16:04,384: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-03-21 13:11:54,680: INFO: config: PyTorch version 2.2.1 available.]
